code/modules/events/immovable_rod.dm

```diff
@@ -72,6 +72,10 @@ In my current plan for it, 'solid' will be defined as anything with density == 1
 	var/dnd_style_level_up = TRUE
 	/// Whether the rod can loop across other z-levels. The rod will still loop when the z-level is self-looping even if this is FALSE.
 	var/loopy_rod = FALSE
+	//SKYRAT ADDITION - RODS DEAL LESS DAMAGE TO MOBS LAYING DOWN
+	/// The percentage of maxhp to apply as damage to a mob laying down on Bump
+	var/maxhp_damage_resting = 0.80
```

Comment:
Why isn't this a define?

Reply:
to allow adminbus if people decide they want to just lay under the rod and pretend to be invincible

Reply:
vars can use defines